core/renderers/geras/path_object.ts

```diff
@@ -90,6 +90,10 @@ export class PathObject extends BasePathObject {
   override applyColour(block: BlockSvg) {
     this.svgPathLight.style.display = '';
     this.svgPathDark.style.display = '';
+    if (!this.style.colourTertiary) {
+      throw new Error(
+          'The renderer did not properly initialize the block style');
```

Comment:
Maybe specify colourTertiary/colourSecondary in the messages to make it easier to track down?

Reply:
Done!